docs/distributed_scenarios.rst

```diff
@@ -0,0 +1,332 @@
+Running distributed scenarios
+=============================
+
+Mahiru is a distributed system. While that doesn't stop anyone from running a
+complete Mahiru system on a single machine, some experiments require setting up
+Mahiru on actual different machines to get realistic results.
+
+So, we need some way of running distributed experiments: we obtain some
+(physical or virtual) machines, install a Mahiru registry on one of them and
+some sites on others, and run some kind of distributed processing scenario.
+Mahiru has a set-up for this, and this page explains how to work with it.
+
+Installing software
+-------------------
+
+The distributed set-up uses `Vagrant <https://vagrantup.com>`_ and `Ansible
+<https://ansible.com>`_ to orchestrate the experiment, and we'll also need
+`Docker <https://docker.com>`_ to build some container images. I'm running this
+on Ubuntu Linux, so that's what these instructions are for. If you have a
+different OS, then you'll probably have to do some figuring-out to get things to
+work.
```

Comment:
Can we make this a bit more non-linux-user friendly? Maybe add some pointers on where to start (i.e. point to the troubleshooting at the end of the file). And add something like 'If you are stuck, please file an issue so we can help you, see CONTRIBUTING.md'